### DISCUSSION TITLE
[Ente Auth] Periodically ask password as memory aid

### DISCUSSION THREAD

**Discussion by martijnkriebel** [ente-io/ente#3618] [Enhancements]
When using PIN/biometrics to lock the app, the user is never asked for the main password that encrypts the 2FA data. It would be good to ask for this password periodically (e.g. every couple days) to prevent users from forgetting it.

**Comment by crKRoEYS** [comment]
But this can be annoying for some people, e.g. if you already use a password manager. So pls with Opt-out.
